### DISCUSSION TITLE
Failed to load resource: net::ERR_INCOMPLETE_CHUNKED_ENCODING

### DISCUSSION THREAD

**Discussion by batthias** [requarks/wiki#4737] [Help / Questions]
I created a _wikijs_ instance [on my Ubuntu server](http://ipmdb.karotte.io/) and it used to work fine.
But now the generated assets are not loading anymore.

`Failed to load resource: net::ERR_INCOMPLETE_CHUNKED_ENCODING` for `/_assets/js/vendor.js?1632539026:1`

And the output is clearly cut off somewhere in the middle. I cannot find any mention of this online, so I have no idea how to even approach the issue.
I am using _node v16.13.0_ but also tried it with a different version before and it is the same.

**Comment by Phil-hostwiki** [comment]
Similar to https://github.com/Requarks/wiki/discussions/4720

**Reply by batthias** [reply]
unfortunately I cannot understand what is supposed to be the answer there

**Comment by Phil-hostwiki** [comment]
In their case, the root file system was filed up.

From this [Stackoverflow](https://stackoverflow.com/questions/29894154/chrome-neterr-incomplete-chunked-encoding-error)  link, it mostly had to do with Antivirus and Permissions issue.
